src/arb/ArbTypes.jl

```diff
@@ -35,6 +35,21 @@ mutable struct arf_struct
   size::UInt # mp_size_t
   d1::UInt # mantissa_struct
   d2::UInt
+
+  function arf_struct()
+    z = new()
+    ccall((:arf_init, libarb), Nothing, (Ref{arf_struct}, ), z)
+    finalizer(_arf_clear_fn, z)
+    return z
+  end
+
+  function arf_struct(exp::Int, size::UInt, d1::UInt, d2::UInt)
+    new(exp, size, d1, d2)
+  end
```

Comment:
This is a very dangerous constructor. If it is not begin used, it should be removed.